src/wp-includes/vars.php

```diff
@@ -69,6 +69,8 @@
 		$is_lynx = true;
 	} elseif ( str_contains( $_SERVER['HTTP_USER_AGENT'], 'Edg' ) ) {
 		$is_edge = true;
+	} elseif ( str_contains( $_SERVER['HTTP_USER_AGENT'], 'Opera' ) || str_contains( $_SERVER['HTTP_USER_AGENT'], 'OPR' ) ) {
```

Comment:
```suggestion
	} elseif ( str_contains( $_SERVER['HTTP_USER_AGENT'], 'Opera' ) || str_contains( $_SERVER['HTTP_USER_AGENT'], 'OPR/' ) ) {
```

I'm wondering if it would be safer to include the slash as that appears to be in the UA string according to the support ticket logged on the beta tester plugin https://wordpress.org/support/topic/bug-opera-doesnt-get-correctly-recognized-as-user-agent/

Reply:
Based on [this article from Opera](https://dev.opera.com/blog/opera-user-agent-strings-opera-15-and-beyond/), it looks like `OPR/` should work, and doesn't look like there's any `OPR Mini/` UA strings out there.

Reply:
Added in [0377829](https://github.com/WordPress/wordpress-develop/pull/4721/commits/037782962c7822f82194e58456850a378a16fecd).